### ISSUE TITLE
Create API endpoint for documents 

### ISSUE THREAD

**Issue by Scarletto** [svthalia/concrexit#3030]
### Describe the change
Create API endpoint for documents.

### Motivation
More useful endpoints is better.
Right now, one can only fetch a document URL by requesting the entire related activity.

**Comment by ColonelPhantom** [comment]
Given that #3050 is done, does this still need to be implemented as well?

I can imagine that maybe a document API would be useful if the app were to get a document page? And otherwise supporting it would be nice in general, but if there's no immediate need we can at least bring down the priority of this issue.

**Comment by DeD1rk** [comment]
> Given that #3050 is done, does this still need to be implemented as well?
> 
> I can imagine that maybe a document API would be useful if the app were to get a document page? And otherwise supporting it would be nice in general, but if there's no immediate need we can at least bring down the priority of this issue.

I agree there's no hurry. There are plenty of higher priority things to add to the app: partners/vacancies and facedetection.
